src/python/TaskWorker/Worker.py

```diff
@@ -97,6 +97,14 @@ def processWorkerLoop(inputs, results, resthost, resturi, procnum, logger):
                     logger.exception('Traceback follows:')
         t1 = time.time()
         logger.debug("%s: ...work on %s completed in %d seconds: %s", procName, task['tm_taskname'], t1-t0, outputs)
+
+        try:
+            out, _, _ = executeCommand("ps u -p %s | awk '{sum=sum+$6}; END {print sum/1024}'" % os.getpid())
```

Comment:
ah.. even more to the point then ! Let's hand adding of psutil to Bruno and Lina as a generally useful thing, and be happy with what you have done for next N months.